init.lua

```diff
@@ -597,6 +597,32 @@ end
 -- NOITA CALLBACKS
 ----------------------------------------------------------------------------------------------------
 
+local slow_position_update_timer = 0
+local old_x = 0
+local old_y = 0
+local function UpdatePlayerPoptrackerPosition()
+	local player = get_player()
+	if not player or player == 0 then return end
+
+	slow_position_update_timer = slow_position_update_timer + 1
+	if slow_position_update_timer < 60 * 10 then return end	-- 10 seconds
+
+	-- Use x/y chunk to reduce traffic
+	local x, y = EntityGetTransform(player)
+	x = math.floor(x / 512)
+	y = math.floor(y / 512)
+
+	if x ~= old_x or y ~= old_y then
+		slow_position_update_timer = 0	-- only reset the timer if we're sending a packet
+		old_x = x
+		old_y = y
+
+		local key = "Noita_position_" .. ap:get_slot()
```

Comment:
```suggestion
		local key = "Noita_position_" .. current_player_slot
```
Usually we use the slot number rather than name for stuff like this